### PR TITLE
refactored the use of local banker

### PR DESCRIPTION
**Pull request by mikeburkat**
a split banker which chooses based on a set of campaigns if the master or local banker will be used.

### CODE REVIEW

**Comment by oktal** [comment]
Two tests are failing. Did you check ?

File: rtbkit/core/banker/split_banker.h [hunk 1]
```diff
@@ -0,0 +1,121 @@
+
```
Comment (oktal) [inline rtbkit/core/banker/split_banker.h]: I think it would be good to put a header on top of the file

Comment (mikeburkat) [inline rtbkit/core/banker/split_banker.h]: yeah i should be better at remembering to put that copyright there.

**Comment by oktal** [comment]
Looks good to me, +1

**Pull request merged**